src/tox/tox_env/python/api.py

```diff
@@ -157,6 +157,14 @@ def extract_base_python(cls, env_name: str) -> str | None:
             return next(iter(candidates))
         return None
 
+    @classmethod
+    def python_spec_for_sys_executable(cls) -> PythonSpec:
+        implementation = sys.implementation.name
+        version = sys.version_info
+        bits = "64" if sys.maxsize > 2**32 else "32"
```

Comment:
bits can be int here

Reply:
Yes, they can but they will be inserted to a string anyway, so I thought why go int to str, when I can use a str literal.